google-cloud-billing-v1/synth.py

```diff
@@ -31,6 +31,7 @@
         "ruby-cloud-title": "Billing V1",
         "ruby-cloud-summary": "Allows developers to manage billing for their Google Cloud Platform projects programmatically.",
         "ruby-cloud-env-prefix": "BILLING",
+        "ruby-cloud-grpc-service-config": "google/cloud/billing/v1/cloud_billing_grpc_service_config.json;google/cloud/billing/v1/cloud_catalog_grpc_service_config.json",
```

Comment:
Is this semi-colon delimiter correct? Could the generator be changed to support an array instead?

Reply:
Yes the semicolon is correct. These values get turned into command line arguments for the Docker-based front-end for the microgenerator (e.g. `--ruby-cloud-grpc-service-config=/path/to/file1.json;/path/to/file2.json`) so we need a string-based delimiter in case of multiple files.